HACKING.md

```diff
@@ -54,6 +54,9 @@ Worker function execution can be tweaked with environment variables. In Google F
 | DYNATRACE_LOG_INGEST_REQUEST_MAX_SIZE | Max size in bytes of single payload to logs ingest endpoint. If it surpasses server limit, payload will be rejected with 413 code | 1048576 (1 mb) |
 | DYNATRACE_LOG_INGEST_EVENT_MAX_AGE_SECONDS | Determines max age of forwarded log event. Should be the same or lower than on cluster | 1 day |
 | GCP_PROJECT | GCP project of log sink pubsub subscription | |
+| USE_PROXY | Depending on value of this flag, function will use proxy settings for either Dynatrace, GCP API or both.
+| HTTP_PROXY | Set the proxy address. To be used in conjunction with USE_PROXY |  |
```

Comment:
is it possible to have both specified? if yes, conditions in helm template may not work corretly

Reply:
yes it is possible and is also recommended. Why? I fix it to cover that case

Reply:
fixed